docs/code-search/code-navigation/how-to/index_other_languages.mdx

```diff
@@ -1,50 +1,67 @@
 # Index other languages
 
-This guide is meant to provide instructions for how to enable precise code navigation for any programming language.
+<p className="subtitle">This guide is meant to provide instructions for how to enable precise code navigation for any programming language.</p>
+
 The general steps for enabling precise code navigation are as follows:
 
-## 1. Install Sourcegraph CLI
+1. Install Sourcegraph CLI
```

Comment:
Same here, the other file was updated and cleaned up for scip and this is getting old content